problem running esphome flasher in lubuntu 20.04

Hi,
I tried to get it to run, installed all dependencies and now I got this traceback when starting :

```
Traceback (most recent call last):
  File "/home/geertsteyaert/esphome-flasher-master/esphome-flasher-master/esphomeflasher/gui.py", line 286, in OnInit
    frame = MainFrame(None, "esphome-flasher (Based on NodeMCU PyFlasher)")
  File "/home/geertsteyaert/esphome-flasher-master/esphome-flasher-master/esphomeflasher/gui.py", line 177, in __init__
    self._init_ui()
  File "/home/geertsteyaert/esphome-flasher-master/esphome-flasher-master/esphomeflasher/gui.py", line 226, in _init_ui
    serial_boxsizer.Add(reload_button, 0, wx.ALIGN_RIGHT, 20)
wx._core.wxAssertionError: C++ assertion "!(flags & wxALIGN_RIGHT)" failed at /home/wxpy/wxPython-4.1.0/ext/wxWidgets/src/common/sizer.cpp(2094) in DoInsert(): Horizontal alignment flags are ignored in horizontal sizers
OnInit returned false, exiting...
```

Any idea?

Thanks.

--- a/esphomeflasher/gui.py
+++ b/esphomeflasher/gui.py
@@ -223,7 +223,7 @@ class MainFrame(wx.Frame):
         serial_boxsizer = wx.BoxSizer(wx.HORIZONTAL)
         serial_boxsizer.Add(self.choice, 1, wx.EXPAND)
         serial_boxsizer.AddStretchSpacer(0)
-        serial_boxsizer.Add(reload_button, 0, wx.ALIGN_RIGHT, 20)
+        serial_boxsizer.Add(reload_button, 0, wx.ALIGN_NOT, 20)
 
         button = wx.Button(panel, -1, "Flash ESP")
         button.Bind(wx.EVT_BUTTON, on_clicked)

